src/decls.js

```diff
@@ -266,3 +259,7 @@ function fixHooks(obj) {
 
     return obj;
 }
+
+function displayName(block, elem) {
+    return `b:${block}${elem ? ` e:${elem}` : ''}`;
```

Comment:
I don't know. I suggested early auto conversion to PascalCase, because we don't know delimiters here.